decidim-comments/app/frontend/comments/comment.fragment.graphql

```diff
@@ -1,7 +1,12 @@
 fragment Comment on Comment {
-  body
-  createdAt
-  author {
-    name
+  ...CommentData
```

Comment:
I included a comment in the backend side https://github.com/AjuntamentdeBarcelona/decidim/blob/b935d9c1b87afd77aee638a6cc62cfd168266ec3/decidim-comments/app/models/decidim/comments/comment.rb#L8

It's not a reason tho but I prefer adding that comment here because I'm not sure if I can include a comment on a `graphql` file.